spec/models/form/sales/questions/ownership_scheme_spec.rb

```diff
@@ -11,6 +11,7 @@
 
   before do
     allow(form).to receive(:start_year_2024_or_later?).and_return(false)
+    allow(form).to receive(:start_year_2025_or_later?).and_return(false)
```

Comment:
Should the 2025 answer options get a test here?

Reply:
Added